### PR TITLE
Add subpackage openQA-munin

### PR DESCRIPTION
**Pull request by perlpunk**
Issue: https://progress.opensuse.org/issues/99741

Tested here:
* OBS https://build.opensuse.org/package/show/home:tinita:branches:devel:openQA/openQA
* Download: https://download.opensuse.org/repositories/home:/tinita:/branches:/devel:/openQA/openSUSE_Tumbleweed/x86_64/

### CODE REVIEW

File: dist/rpm/openQA.spec [hunk 1]
```diff
@@ -288,6 +288,19 @@ upgrading the system if devel:openQA packages are stable and contain updates. It
 is complementary to auto-update which also reboots the system and does updates
 regardless of whether devel:openQA contains updates.
 
+%package munin
+Summary:        Munin scripts
```
Comment (Martchus) [inline dist/rpm/openQA.spec:292]:
```suggestion
Summary:        Munin scripts for monitoring openQA
```

File: dist/rpm/openQA.spec [hunk 2]
```diff
@@ -742,4 +761,14 @@ fi
 %{_unitdir}/openqa-continuous-update.*
 %{_datadir}/openqa/script/openqa-continuous-update
 
+%files munin
+%defattr(-,root,root)
+%doc contrib/munin/config/minion.config
+%dir %{_prefix}/lib/munin
+%dir %{_prefix}/lib/munin/plugins
```
Comment (Martchus) [inline dist/rpm/openQA.spec:768]: This is not a usual library. Should it really go under `lib`? Have you checked whether there are already other packages containing Munin plugins and what location those packages use?

Comment (perlpunk) [reply dist/rpm/openQA.spec:768]: At least that's the location where the plugins are located that come with munin itself...
There is one plugin I could find that uses `_datadir`: https://build.opensuse.org/package/view_file/security:Zorp:7:0:7/zorp/zorp-7_0-7.spec?expand=1

**Review by okurz — APPROVED** [review]

File: dist/rpm/openQA.spec (hunk 2 above)
Comment (Martchus) [reply dist/rpm/openQA.spec:768]: Looks like the munin package in Factory uses this path, indeed. I suppose then it is fine.

**Review by Martchus — APPROVED** [review]

File: dist/rpm/openQA.spec (hunk 1 above)
Comment (perlpunk) [reply dist/rpm/openQA.spec:292]: hm, I still wanted to do that before it's merged... :)

**Pull request merged**